src/gui/buttons.py

```diff
@@ -62,8 +118,64 @@ def get_new_missions_handler(event: UIEvent) -> UIEventHandler:
     eng.refresh_mission_board()
 
 
-def get_new_missions_button() -> UIFlatButton:
-    btn = UIFlatButton(text="New Missions")
+def get_new_missions_button() -> UITextureButton:
+    main_tex = PixelatedNinePatch(
+        left=1, right=1, bottom=3, top=1, texture=WindowData.buttons[7]
+    )
+    pressed_tex = PixelatedNinePatch(
+        left=1, right=1, bottom=3, top=3, texture=WindowData.buttons[9]
+    )
+    hovered_tex = PixelatedNinePatch(
+        left=1, right=1, bottom=3, top=1, texture=WindowData.buttons[11]
+    )
+    btn = UITextureButton(
+        text="New Missions",
+        texture=main_tex,
+        texture_hovered=hovered_tex,
+        texture_pressed=pressed_tex,
+    )
     btn.on_click = get_new_missions_handler
 
     return btn
+
+
+def end_turn_handler(view) -> UIEventHandler:

```

Comment:
This function is not the handler, the function it returns (`_handle`) is the handler, so this is a handler getter!

```suggestion
def get_end_turn_handler(view) -> UIEventHandler:

```